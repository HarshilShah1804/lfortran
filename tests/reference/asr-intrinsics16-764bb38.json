{
    "basename": "asr-intrinsics16-764bb38",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/intrinsics16.f90",
    "infile_hash": "356a312a537f31cd442f72d77ec775eacfd9fe68d90ca06c009460bd",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-intrinsics16-764bb38.stderr",
    "stderr_hash": "d41a66ae628695f61956ae964c3501bf958879c53d2c026984c7077c",
    "returncode": 2
}