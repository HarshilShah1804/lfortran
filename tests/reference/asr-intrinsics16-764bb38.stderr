semantic error: arguments of `repeat` intrinsic must be scalar
 --> tests/errors/intrinsics16.f90:2:38
  |
2 |     character(3), parameter :: ar1 = repeat(["abc", "#^1", "123"], [1, 2, 3])
  |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
